test/models/concerns/json_fields_parsable_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "test_helper"
```

Comment:
@Pralish can we rename this file to _test?

Reply:
Sure @donrestarone . Habits😅